src/main/java/org/opensearch/sdk/SearchExtension.java

```diff
@@ -843,18 +843,25 @@ public P getParser() {
      * Context available during fetch phase construction.
      */
     class FetchPhaseConstructionContext {
-        private final Map<String, Highlighter> highlighters;
+
```

Comment:
We should leave the Extension interfaces exactly matching their plugin counterparts whenever possible.

Also this falls into the same category of my previous question, adding a setter that's not needed actually reduces the encapsulation and benefit of a final field.  Was this a spotbugs issue?